models/usage.go

```diff
@@ -0,0 +1,244 @@
+package models
+
+import (
+	"errors"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+
+	"github.com/jinzhu/gorm"
+)
+
+// DataUsageTier is a type of usage tier
+// which governs the price per gb ratio
+type DataUsageTier string
+
+// String returns the value of DataUsageTier as a string
+func (d DataUsageTier) String() string {
+	return string(d)
+}
+
+// PricePerGB returns the price per gb of a usage tier
+// rtc indicates whether or not its an RTC based paymetn
+func (d DataUsageTier) PricePerGB() float64 {
+	switch d {
+	case Light:
+		return 0.22
+	case Plus:
+		return 0.165
+	case Partner:
+		return 0.16
+	default:
+		// this is a catch-all for free tier
+		// free tier users will never encounter a charge call
+		return 9999
```

Comment:
`-1` is the standard way to do this i think? doesnt matter that as long as we're careful tho

Reply:
Good call